File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/component/barrage/view/adapter/BarrageItemAdapter.java
package com.trtc.uikit.livekit.component.barrage.view.adapter;

import android.view.ViewGroup;
import androidx.recyclerview.widget.RecyclerView;

import com.trtc.uikit.livekit.component.barrage.store.model.Barrage;

public interface BarrageItemAdapter {
    RecyclerView.ViewHolder onCreateViewHolder(ViewGroup parent);

    void onBindViewHolder(RecyclerView.ViewHolder holder, int position, Barrage barrage);
}

File: Android/tuilivekit/src/main/java/com/trtc/uikit/livekit/component/barrage/view/adapter/BarrageItemAdapter.kt
package com.trtc.uikit.livekit.component.barrage.view.adapter

import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import io.trtc.tuikit.atomicxcore.api.Barrage

interface BarrageItemAdapter {
    fun onCreateViewHolder(parent: ViewGroup): RecyclerView.ViewHolder

    fun onBindViewHolder(holder: RecyclerView.ViewHolder, position: Int, barrage: Barrage)
}
